toolkit/imageconfigs/core-fips.json

```diff
@@ -46,7 +46,13 @@
                 "packagelists/hyperv-packages.json",
                 "packagelists/fips-packages.json",
                 "packagelists/core-packages-image.json",
-                "packagelists/cloud-init-packages.json"
+                "packagelists/cloud-init-packages.json",
+                "packagelists/legacy-bootloaders.json"
+            ],
+            "PostInstallScripts": [
+                {
+                    "Path": "postinstallscripts/core-legacy/postinstall-cleanup-packages.sh"
+                }
```

Comment:
Could you add a more robust explanation of why `grub2 needs to be installed into some of these packages and then separately removed through a script? It's hard to understand the reason behind this change and I'm curious if it isn't a feature we're missing in our tooling, that we're now trying to fix with a hacky workaround.